test/functional/wallet_listdescriptors.py

```diff
@@ -134,6 +134,26 @@ def run_test(self):
         }
         assert_equal(expected, wallet.listdescriptors())
 
+        self.log.info('Test taproot descriptor do not have mixed hardened derivation marker')
```

Comment:
Since the issue is not specific to taproot descriptors and instead the ones containing MiniScript expression.

```diff
-        self.log.info('Test taproot descriptor do not have mixed hardened derivation marker')
+        self.log.info('Test descriptors do not have mixed hardened derivation marker')
```